solvers/mosek/mosekmodelapi.h

```diff
@@ -56,11 +56,11 @@ class MosekModelAPI :
   /// and have an implementation,
   /// or a conversion rule is needed in a derived FlatConverter
   ACCEPT_CONSTRAINT(LinConLE, Recommended, CG_Linear)
-  void AddConstraint(const LinConLE& lc);
+  void AddConstraint(const LinConLE& lc) {}
```

Comment:
This won't work, LinCon/... are necessary

Reply:
Only the AddConstraint with LinConRange argument is called, so we decided to handle the bound key logic there. But if you suggest a better way, I would rewrite. So that for an LE constraint, AddConstraint(const LinConLE& lc) is called.

Reply:
For models with logical constraints, redefinitions would create LE/EQ/GE constraints. While it is possible to write redefinitions from them into Range, which would be invoked if the LE/EQ/GE were not accepted, tis would add unnecessary transformations. I suggest implementing these methods, best by adding Mosek rhs constraints of the corr. type